library/src/main/java/io/ona/kujaku/services/MapboxOfflineDownloaderService.java

```diff
@@ -110,9 +111,12 @@
     private long currentMapDownloadId;
 
     private DownloadProgressNotification downloadProgressNotification;
-    public static final int PROGRESS_NOTIFICATION_ID = 85;
+    public static final int PROGRESS_NOTIFICATION_ID = 80;
     public static final int REQUEST_ID_STOP_MAP_DOWNLOAD = 1;
-    public int LAST_DOWNLOAD_COMPLETE_NOTIFICATION_ID = 87;
+
+    // Should persist across service instances but within the same app session
+    public static int LAST_DOWNLOAD_COMPLETE_NOTIFICATION_ID = 180;
```

Comment:
Is the assumption here that the DownloadCompleteNotification will not be shown more that 100 times within the app's life-time?

Reply:
> Is the assumption here that the DownloadCompleteNotification will not be shown more that 100 times within the app's life-time?

Within the application lifecycle not the life-time

Reply:
Yea, that's what I meant.

So if the application is not killed for a while, will this ever be an issue? Could the 100 limit ever be crossed?

Reply:
**Could a host application successfully download 100 maps within one application life-cycle?** Realistically not, but possibly yes.

I'll increase the gaps to 1000. Is that ok?

Reply:
That's fine.